variable/include/scipp/variable/bucket_variable.tcc

```diff
@@ -106,10 +106,12 @@ public:
 template <class T> class BinVariableMaker : public BinVariableMakerCommon<T> {
 private:
   const VariableConstView
-  bucket_parent(const scipp::span<const VariableConstView> &parents) const {
+  bin_parent(const scipp::span<const VariableConstView> &parents) const {
+    if(parents.empty())
```

Comment:
clang-format?

Hmm, it looks like our CI regex does not match `tcc`.

Reply:
I'll update https://github.com/scipp/scipp/blob/master/.azure-pipelines/templates/code_quality_checks.yml#L22

Reply:
On second thoughts - applied fix to changed file here, but opening https://github.com/scipp/scipp/pull/1545 separately as will involve updating a number of unrelated tcc files I suspect.